catalog/spatial/geocoding/spatial-geocoding-geocoder/src/test/java/org/codice/ddf/spatial/geocoder/GazetteerGeoCoderTest.java

```diff
@@ -179,14 +179,4 @@ public void testGetCountryCodeParseException() throws ParseException, GeoEntryQu
     countryCode = gazetteerGeoCoder.getCountryCode(TEST_POINT, 50);
     assertThat(countryCode.isPresent(), is(false));
   }
-
-  @Test
-  public void testGetCountryCodeInvalidCountryCode() throws ParseException, GeoEntryQueryException {
-    when(geoEntryQueryable.getCountryCode(TEST_POINT, 50))
-        .thenReturn(Optional.of("not a country code"));
```

Comment:
I think this test should be left here; it's a valid use case. Sorry if my original comment wasn't clear, I just meant that your previous change from `"not a country code"` to `Optional.empty()` should be reverted.

Reply:
i am trying to debate whether this is still a valid case. Previously GazetteerGeoCoder.getCountryCode() attempts to convert ISO2 to ISO3, for a case  of "not a country code", it will fail the conversion and hence return a Option.empty() and pass the test.
Now with the simplified logic, GazetteerGeoCoder.getCountryCode() just forward the country code from GazetteerQueryCatalog.getCountryCode(). Then assertThat(countryCode.isPresent(), is(true)) instead of false. which is a strange test case if we keep it.

Reply:
perhaps, GazetteerGeoCoder.getCountryCode() should do some validation?

Reply:
Ah you are right, I think this test can be removed then 👍 .

As far as validation goes... at the end of the day, `GazetteerQueryCatalog.getCountryCode()` just returns an `Optional` of the `Location.COUNTRY_CODE` attribute. We have a validator that checks if the value is in alpha3 format but it doesn't actually guarantee it will be. If this is only for gazetteer data though, we might be ok.